platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ExportMigrationConfigurationAdminEntry.java

```diff
@@ -55,7 +58,19 @@ public boolean store() {
     if (!stored) {
       LOGGER.debug(
           "Exporting configuration [{}] to [{}]...", configuration.getPid(), entry.getPath());
-      this.stored = entry.store((r, out) -> persister.write(out, configuration.getProperties()));
+      final Dictionary<String, Object> props = configuration.getProperties();
```

Comment:
❓Instead of adding this logic, should we consider storing the content of the `Dictionary` as-is and store the factory PID, service PID and bundle location as separate attributes of the entry? This would keep all the values in-sync in case there was a good reason for the difference between what's in the `Dictionary` and what the methods return.

Reply:
I don't think we need to add that complexity. The fact is that these 2 properties and the felix install file re never re-imported on purpose. This follows the paradigm already used by Felix in both the config installer and cellar. As such, it simplifies to simply re-use these properties like this.